src/apscheduler/datastores/mongodb.py

```diff
@@ -294,7 +292,9 @@ async def acquire_schedules(self, scheduler_id: str, limit: int) -> list[Schedul
                     }
                     self._schedules.update_many(filters, update, session=session)
 
-        return schedules
+            return schedules
```

Comment:
Why indent this code?

Reply:
Reverting, but this was the same idea as below.